tests/beaker_tests/cisco_aaa_authentication_login/test_aaaauthelogin_nondefaults.rb

```diff
@@ -97,12 +96,12 @@
 
     tests[id][:code] = [0, 2]
```

Comment:
I know this is not part of your changes, but I would expect the exit code to be 2 each time for this test if the test is cleaning up properly in the setup for the test.